Add_user.css

```diff
@@ -20,52 +27,62 @@ body{
 
 nav{
 	width:100%;
-	background-color: white;
+	background-color: #0C59CF;
 	overflow:auto;
 	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
-
+/* Set up position for the text in nav to the right */
 li{
-	float:left;
+	float:right;
 }
 
 
-
+/* Set up Drop down menu for */
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
 ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 130px;
 	width: 100px;
 	display: none;
 	visibility: hidden;
 	z-index: 5;
+}
 
+ul li #a{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 5;
 }
+
 	
 ul li ul li{
-	background-color: purple;
+	background-color:  #0C59CF;
```

Comment:
This will make the background color is I drop down menu column to be navy blue you hover on the like that that column is belong to. This is match with the nav bar  and by dong this it success my aim to make the music database to be consistence.